mlbench_core/utils/pytorch/topology.py

```diff
@@ -3,22 +3,34 @@
 import torch.distributed as dist
 
 
+def get_backend_tensor(tensor):
```

Comment:
We should also call this function for the transformations in `distributed.py` and `partition.py` to avoid code repetition.